bootstrap-joining-demo/kubernetes-api/src/main/resources/application.conf

```diff
@@ -6,10 +6,28 @@ akka {
 
   discovery {
     # pick the discovery method you'd like to use:
-    method = kubernetes-api
```

Comment:
I just repurposed the kubernetes-api demo for my purposes.

Reply:
Thief!